ts_src/confidential.ts

```diff
@@ -83,7 +81,19 @@ export function unblindOutput(
   };
 }
 
-export function unblindWitnessUtxo(
+export function unblindOutput(
```

Comment:
this shouldn't be exported anymore and should accept an output and a nonce (it's actually the raw implementation of the unblindOutputWithNonce exported func)